libraries/c_sdk/standard/mqtt/src/private/iot_mqtt_internal.h

```diff
@@ -913,4 +735,146 @@ bool _IotMqtt_GetNextByte( void * pNetworkConnection,
 void _IotMqtt_CloseNetworkConnection( IotMqttDisconnectReason_t disconnectReason,
                                       _mqttConnection_t * pMqttConnection );
 
+/*----------------- MQTT Serailization /Deserialization Wrapper functions for Shim------------------*/
+
+/**
+ * @brief The wrapper for  @ref _IotMqtt_SerializeConnect
+ *
+ * See @ref _IotMqtt_SerializeConnect for a
+ * description of the parameters and return values.
+ */
+IotMqttError_t _connectSerializeWrapper( const IotMqttConnectInfo_t * pConnectInfo,
```

Comment:
For any new code we should avoid _ at the start.  You can rename them as connectSerializeWrapper()